### ISSUE TITLE
Automated A11y test results

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#349]
# Summary

Testing of site, page by page, with Axe Chrome Extension (which tests to WCAG 2.1 AA) to pick up on further issues.

# Results

No issues for any of the main pages except the sitemap

- [ ] sitemap needs to have language specified. 
- [ ] News items are missing image alt-text

# Details

## Main Pages

- [ ] Homepage
- [ ] Who We Are
- [ ] What we do
- [ ] How We Work
- [ ] Our Team
- [ ] Contact
- [ ] Info For Editors
- [ ] Info for readers and authors
- [ ] Info for Libraries and consortia
- [ ] Open Access movement
- [ ] resources
- [ ] OA Glossary
- [ ] Reading List
- [ ] Internships
- [ ] Journals
- [ ] Announcements and Blog
- [ ] newsletters and annual reports
- [ ] log in
- [ ] register
- [ ] publisher policies
- [ ] privacy
- [ ] accessibility
- [ ] janeway
- [ ] copyright
- [ ] sitemap
    - the language is not set on this page

### news pages - e.g. https://www.openlibhums.org/news/579/
- missing alt-text
- link does not have discernible text 
```
<p><i><a href="https://www.glossa-journal.org/" target="_blank">Glossa</a></i><a href="https://www.glossa-journal.org/" target="_blank"> </a>publishes about 130 articles/year, so the work is rather demanding on a team of volunteer editors. A flawless and flexible submission system is a necessity. Janeway delivers.<br></p> 
```
link appears twice, once without text.

Likewise on https://www.openlibhums.org/news/425/

```
<p>If you like the work that the Open Library of Humanities is doing, please consider asking your institution to support us financially. We cannot operate without our library members. More details for libraries can be found here: <a href="https://www.openlibhums.org/plugins/supporters/signup" target="_blank">https://www.openlibhums.org/plugins/supporters/signup</a><a href="https://www.openlibhums.org/plugins/supporters/signup" target="_blank" style="background-color: rgb(255, 255, 255);"></a></p>
```
